### ISSUE TITLE
slow meteor build time

### ISSUE THREAD

**Issue by trysenga** [TAPevents/tap-i18n#213]
I'm looking into the extreme slowness of our build and when profiling this caught my eye:

| â””â”€ bundler.bundle...........................................133,010 ms (1)
|    â””â”€ files.withCache.......................................133,010 ms (1)
|       â”œâ”€ compiler.compile(the app)..........................112,114 ms (1)
|       â”‚  â””â”€ files.withCache.................................112,114 ms (3)
|       â”‚     â””â”€ compileUnibuild (the app)....................112,114 ms (3)
|       â”‚        â”œâ”€ PackageSource#_findSources....................479 ms (3)
|       â”‚        â”‚  â””â”€ files.withCache............................479 ms (3)
|       â”‚        â”‚     â”œâ”€ files.realpath                          187 ms (244)
|       â”‚        â”‚     â”œâ”€ files.readdir                           111 ms (488)
|       â”‚        â”‚     â”œâ”€ files.stat                               90 ms (440)
|       â”‚        â”‚     â””â”€ other files.withCache                    91 ms
|       â”‚        â”œâ”€ files.stat                                     88 ms (405)
|       â”‚        â”œâ”€ files.readFile                                479 ms (636)
|       â”‚        â”œâ”€ sha1                                          151 ms (636)
|       â”‚        â”œâ”€ legacy handler (.i18n.json)               109,909 ms (39)
|       â”‚        â””â”€ other compileUnibuild (the app)               976 ms

Over a minute spent in legacy handler for .i18n.json. Is it possible we've mis-configured our use of tapi18n or does it exponentially slow down when building multiple languages? Our lang files have 2200 lines and we have 13 supported languages.

**Comment by MrSpark2591** [comment]
We are hacing similar issue

`Top leaves:
| other legacy handler (.i18n.json).......................20,188 ms (20)
| other compileUnibuild (the app)............................886 ms (4)
| files.readFile.............................................165 ms (176)
| other ClientTarget#write...................................144 ms (1)
| other Builder#copyNodeModulesDirectory.....................133 ms (43)
| other JsImage#write........................................106 ms (1)
| 
| (#11) Total: 22,801 ms (Rebuild App)
`

**Comment by trysenga** [comment]
I figured it out - it is kicking off the legacy handler for any minor file change. For us, we dropped using this package and wrote our own. Its possible that with webpack or a different build process the times would improve.

**Comment by MrSpark2591** [comment]
@trysenga We have a really big project using this package now. 
Any good documentation to switch to webpack? We are using blaze.

**Comment by trysenga** [comment]
No, we switched and switched back due to some version compatibility problems. If you go through the effort, make sure you _know_ it will really fix the issue. Webpack did make UI development nicer (faster local builds when changing html or css). We will be switching back now that we are upgrading our meteor version in the next few months.

**Comment by MrSpark2591** [comment]
Ok, I will try to document the changes so that help out the community.
